### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by rakelkar**
add apache2 license as requested in #8

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (jroggeman) [inline LICENSE]: Should add in the year and your name here :)

Comment (jroggeman) [reply LICENSE]: wait nevermind, apache license is a lot shorter than this, think this is the full text that isn't needed

Comment (jroggeman) [reply LICENSE]: LICENSE only needs to be this line and below with the year and owner replaced, as original full license is linked in the short text

Comment (rakelkar) [reply LICENSE]: github added the license for me automatically - i didnt edit it manually... are you saying I should?

**Comment by rakelkar** [comment]
@jroggeman updated!

**Pull request merged**